tests/templates/kuttl/smoke/30-install-zk.yaml.j2

```diff
@@ -16,6 +16,12 @@ spec:
     config:
       logging:
         enableVectorAgent: {{ lookup('env', 'VECTOR_AGGREGATOR') | length > 0 }}
+      resources:
```

Comment:
My recommendation would be to remove this lines, so that we implicitly test the default zk resources (are they enough?)

Reply:
done